### DISCUSSION TITLE
Open HUD with mouse

### DISCUSSION THREAD

**Discussion by Misamangame** [Courseplay/Courseplay_FS22#3101] [General]
Hello, is there some specific reason why open HUD with mouse is in vehicle settings?  Maybe it is just my opinion but I would like to see it in global settings  as for me this should be set for everyone same and it is a bit of hustle to change it on each new vehicle.

**Comment by Tensuko** [comment]
No, it belongs into vehicle.
Reason: turn off when have frontloader or anything else!

If want that off global just remove the keybind in the ingame menu for FS22.

**Reply by Misamangame** [reply]
"If want that off global just remove the keybind in the ingame menu for FS22"
Good idea did not thought about that, thx
